src/Integration.Vsix/SonarLintDaemon/SonarLintDaemonInstaller.xaml.cs

```diff
@@ -48,65 +51,96 @@ public SonarLintDaemonInstaller(ISonarLintSettings settings, ISonarLintDaemon da
             {
                 throw new ArgumentNullException(nameof(daemon));
             }
+            if (logger == null)
+            {
+                throw new ArgumentNullException(nameof(logger));
+            }
 
             this.settings = settings;
             this.daemon = daemon;
+            this.logger = logger;
 
             InitializeComponent();
         }
 
         private void Window_ContentRendered(object sender, EventArgs args)
         {
-            ProgressBar.Visibility = Visibility.Visible;
-            CompletedMessage.Visibility = Visibility.Collapsed;
+            try
+            {
+                ProgressBar.Visibility = Visibility.Visible;
+                CompletedMessage.Visibility = Visibility.Collapsed;
+
+                this.logger.WriteLine(Strings.Daemon_DownloadingDaemon);
```

Comment:
I'll move this logging statement and the one in line 76 - they don't make sense where they are.